integration/commands_freemonitoring_test.go

```diff
@@ -111,7 +111,7 @@ func TestCommandsFreeMonitoringSetFreeMonitoring(t *testing.T) {
 				err := s.Collection.Database().RunCommand(s.Ctx, bson.D{{"getFreeMonitoringStatus", 1}}).Decode(&actual)
 				require.NoError(t, err)
 
-				actualStatus, ok := actual.Map()["state"]
+				actualStatus, ok := actual.Map()["debug"].(bson.D).Map()["state"]
```

Comment:
Let's make `GOFLAGS='-run=TestCommandsFreeMonitoringSetFreeMonitoring' task test-integration-mongodb` and `GOFLAGS='-run=TestCommandsFreeMonitoringSetFreeMonitoring'  task test-integration-pg` to both pass.

What we could do is that, it can check `actualStatus, ok := actual.Map()["state"]`, and if it's disabled and has `debug` field, we can check debug field to see if it is `enabled`.

We want to make sure that change also works for current version of MongoDB until we update it :hugs: 

Let me know if you need any help :hugs:

Reply:
Hi! I've made the changes. Not too sure about my approach, let me know if it's okay. Thanks!

Reply:
I like how you did it! I have minor assertion suggestion in another comment